pages/complete-todo.ts

```diff
@@ -0,0 +1,46 @@
+import {Locator, Page}  from '@playwright/test';
+
+export class CompletedPage {
+    
+    private page: Page; 
+    private completeBtn : Locator;
+    private completedTab : Locator;
+    private completedTime : Locator;
+    private completedItems : Locator;
+    private toDoTab : Locator ;
+
+
+
+    constructor(page) {
+        
+        this.page = page;
+        this.completeBtn = page.getByTestId('complete-button')
+        this.completedTab = page.getByTestId('completed-tab')
+        this.toDoTab = page. getByTestId('todo-tab')
+        this.completedTime = page.getByTestId('task-completed-time')
+        
+    }
+
+
+async goto() {
+    await this.page.goto('http://localhost:3000/');
+  }
+
+async completeTodo () {
+    await this.toDoTab.click();
+    await this.completeTaskFromToDo(0).click();
```

Comment:
Consider passing index to the function `complete Todo`, otherwise you will always complete only first todo